toolkits/landcover/impl/bands.js

```diff
@@ -71,7 +71,7 @@ var SPECTRAL_INDEX_EXPRESSIONS = {
  *
  * @param {ee.Image} image The source image.
  * @param {!Array<string>} indices The list of indices to calculate.
- * @return {ee.Image} The updated image.
+ * @return {!Array<!ee.Image>} The new spectral index bands.
```

Comment:
This is technically correct now, but not what's supposed to happen.

This should really return an Image, which means the map at the bottom (which is mapping over a regular javascript array) should be wrapped with an ee.Image constructor.  (Also the docs say it "adds" to the existing image; that's not true either, it should say: "Compute the specified..."